keeper/abci.go

```diff
@@ -1,80 +1,187 @@
 package keeper
 
 import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
 	abci "github.com/cometbft/cometbft/abci/types"
+	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/vitwit/avail-da-module/types"
 )
 
+type StakeWeightedVotes struct {
```

Comment:
for this type of structs,let's maintain a seperate file(types.go) and move these structs to there.